asciidoc/courses/genai-fundamentals/modules/3-knowledge-graphs/lessons/1-what-is-a-knowledge-graph/lesson.adoc

```diff
@@ -66,7 +65,7 @@ The organizing principles describing the content in GraphAcademy could look like
 image::images/org-principles.svg[A Graph showing 4 types of content]
 
 [.slide.col-40-60]
-== Organizing principles
+== Organizing principles <- Why is this repeated?
```

Comment:
I'm familiar with the asciidoc syntax used here, so I can't really say whether this header is repeated due to some rendering reason, but it seems like both paragraphs say approximately the same thing.

Reply:
It should have `.discrete` added to the slide. e.g.

```
[.slide.discrete.col-40-60]
```

This tells GraphAcademy that it should be a different slide (with the same title), but the title shouldn't be shown in the text (i.e. its discrete)

Reply:
Ah, makes more sense! Thanks!